StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/Vertical Saved Payment Method Screen/VerticalSavedPaymentMethodsViewController.swift

```diff
@@ -70,12 +70,15 @@ class VerticalSavedPaymentMethodsViewController: UIViewController {
     }
 
     private var headerText: String {
+        let nonCardPaymentMethods = paymentMethods.filter({ $0.type != .card })
+        let onlyCards = nonCardPaymentMethods.isEmpty
```

Comment:
I typically like to name booleans in a way that answers a yes or no question.. e.g.(`is...`, `has...`, etc) -- e.g. hasOnlyCards